src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -210,18 +209,21 @@
 (defn- unusable-area-height
   []
   (+ (safe-area/get-top)
+     12
      32
      11 ;;top page buttons & button's padding
      (safe-area/get-bottom)))
```

Comment:
I think if this value is related to the safe-area, we should naming it as a constant so that we can change it easily later on.